var/spack/repos/builtin/packages/librsvg/package.py

```diff
@@ -22,15 +22,27 @@ class Librsvg(AutotoolsPackage):
 
     depends_on("gobject-introspection", type="build")
     depends_on("pkgconfig", type="build")
-    depends_on("rust", type="build", when="@2.41:")
+    # rust minimal version from NEWS file and upper bound because "Unaligned
+    # references to packed fields are a hard error" starting from 1.69
+    depends_on("rust@1.40:1.68", when="@2.50:", type="build")
```

Comment:
@manuelakuhn do you know if building v2.56 with rust >1.68 works?

Reply:
It seems to work, at least it librsvg@2.56.2 installs with `--test=root` for me. I have added this version as well.